### DISCUSSION TITLE
Practica Abogabot

### DISCUSSION THREAD

**Discussion by HoracioTorrez** [Launch-X-Latam/MisionFrontEnd#238] [General]
Hola a todos apenas pude ponerme al corriente con la pratica de Abogabot, les dejo el resultado con la esperanza de obtener sus criticas, recomendacions y comentarios saludos desde Hidalgo, México.

https://github.com/HoracioTorrez/Abogabot/blob/main/Abogabot.md

**Comment by iVanOoOvich** [comment]
Te quedó muy bien!

**Reply by HoracioTorrez** [reply]
Muchas Gracias! es la primera vez que hago algo asi jeje

**Comment by Cerecero** [comment]
Lo mejor las imagenes de Better Call Saul 10/10 jaja

**Reply by HoracioTorrez** [reply]
Muchas Gracias. se me ocurrio lo de Saul y no salio tan mal jaja...saludos
